packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -696,12 +716,17 @@ const _indexContent = async (
   }
 
   await sequential(documentPaths, async (filepath) => {
-    const dataString = await database.bridge.get(normalizePath(filepath))
-    const data = parseFile(dataString, path.extname(filepath), (yup) =>
-      yup.object({})
-    )
-    if (database.store.supportsSeeding()) {
-      await database.store.seed(normalizePath(filepath), data, seedOptions)
+    try {
+      const dataString = await database.bridge.get(normalizePath(filepath))
+      const data = parseFile(dataString, path.extname(filepath), (yup) =>
+        yup.object({})
+      )
+      if (database.store.supportsSeeding()) {
+        await database.store.seed(normalizePath(filepath), data, seedOptions)
+      }
+    } catch (error) {
+      console.error(`Unable to seed ${filepath}`)
+      throw error
```

Comment:
Same as the comment above, it seems like the error we throw should be more user-friendly, and we wouldn't need the console log.

Reply:
Fixed